app-shopping/src/main/java/com/shopping/controller/PhoneController.java

```diff
@@ -72,5 +74,17 @@ public Optional<PhoneModel> getById(@PathVariable Long id) throws ProductNotFoun
         }
         return res;
     }
+    @PutMapping("/savemodelno")
+    public ResponseEntity<String> update(@RequestBody  PhoneModel pm)
```

Comment:
what is the difference b/w put and patch?

why this is used here?

Reply:
Patch is used to update the resource while Put is used to update particular entry of resource for example email address...
Its a mistake ill look into it and do the required changes. Thank you